README.md

```diff
@@ -94,10 +94,10 @@ Log into [Azure portal](https://portal.azure.com). **Note**, In order to perform
 2. Fill in required template parameters and click the `Purchase` button to start a deployment. I.e.:
    - `Name` - Any name
    - `Storage Name` - Any Storage Account name (that does not currently exist)
-   - `Alertlogic Access Key Id` - `access_key_id` returned from AIMs [above](#create_an_alert_logic_access_key)
-   - `Alertlogic Secret Key` - `secret_key` returned from AIMs [above](#create_an_alert_logic_access_key)
-   - `Alertlogic API endpoint` - usually `api.global-services.global.alertlogic.com` 
-   - `Alertlogic Data Residency` - usually `default`
+   - `Alert Logic Access Key Id` - `access_key_id` returned from AIMs [above](#create_an_alert_logic_access_key)
```

Comment:
could you please change `Id` to `ID`, requested by Patrick.

Reply:
I deliberately kept it as `Id` because I thought we decided that it was consistent with everything else.  If I make it an exception and called it `ID` won't it look inconsistent and a bit weird?